### PR TITLE
Neutral model perpendicular BC improvements

### PR DESCRIPTION
**Pull request by mikekryjak**
We need the ability to easily define neutral transport at the boundaries. In particular, need an intuitive way to allow neutral density and momentum to escape the core and neutral pressure to escape the SOL. Right now this is complicated because there are hardcoded BCs on both neutral parameters as well as diffusivities (see https://github.com/bendudson/hermes-3/issues/117).

- [x] Expose neutral parameter BCs in the input file
- [x] Expose neutral diffusivity BCs in the input file
- [x] Change neutral BC defaults so that they are Neumann if no user setting is changed (to match what was hardcoded before)

### CODE REVIEW

File: src/neutral_mixed.cxx [hunk 1]
```diff
@@ -31,6 +31,7 @@ NeutralMixed::NeutralMixed(const std::string& name, Options& alloptions, Solver*
   solver->add(Nn, std::string("N") + name);
   solver->add(Pn, std::string("P") + name);
   solver->add(NVn, std::string("NV") + name);
+  /// solver->add(Dnn, std::string("Dnn" + name));
```
Comment (bendudson) [inline src/neutral_mixed.cxx:34]:
```suggestion
```

**Review by bendudson — APPROVED** [review]

File: src/neutral_mixed.cxx (hunk 1 above)
Comment (mikekryjak) [reply src/neutral_mixed.cxx:34]: Thanks @bendudson Not sure why I left that there, or why Dnnd didn't need to be treated this way. I have a feeling that I tried to add it to match the other variables, but then found that it made Hermes-3 crash. What does `solver->add()` do?

Comment (bendudson) [reply src/neutral_mixed.cxx:34]: Hey @mikekryjak `solver->add()` adds the variable to the time integrator (e.g. CVODE), making it a time-evolving quantity. The solver will set the variable value, and expect `ddt()` of that variable to be set.

**Comment by mikekryjak** [comment]
Ignore the previous comment - I got myself confused. I changed the defaults for the neutral diffusion, not the anomalous diffusion. I think that means there's a bug.

**Pull request merged**